tools/cli/vz/cmd/login/login_test.go

```
@@ -0,0 +1,288 @@
+	"github.com/verrazzano/verrazzano/tools/cli/vz/pkg/helpers"
+	"io"
+	"k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	"k8s.io/client-go/kubernetes"
+	k8sfake "k8s.io/client-go/kubernetes/fake"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	"net"
+	"net/http"
+	"net/url"
+	"os"
+	"strconv"
+	"strings"
+	"testing"
+)
+
+var (
+	fakeVerrazzanoAPIURL   = "verrazzano.fake.nip.io/12345"
+	fakeVerrazzanoClientID = "fakeclient"
+	fakeVerrazzanoRealm    = "fakerealm"
+	fakeAuthCode           = "euifgewuhfoieuwhfiuoewhfioewhfoihwfihfgiheriofwerhfiruhgoreihgccccccccccccccccc"
+	test1                  = []struct {
+		args   []string
+		output string
+	}{
+		{
+			[]string{fakeVerrazzanoAPIURL},
+			"Login successful!\n",
+		},
+	}
+	test2 = []struct {
+		args   []string
+		output string
+	}{
+		{
+			[]string{fakeVerrazzanoAPIURL},
+			"Already Logged in\n",
+		},
+	}
+)
+
+var status string
+var codeChallenge string
+
+type TestKubernetes struct {
+	fakeProjectClient  projectclientset.Interface
+	fakeClustersClient clustersclientset.Interface
+	fakev8oCleint      verrazzanoclientset.Interface
+	fakek8sClient      kubernetes.Interface
+}
+
+func (o *TestKubernetes) GetKubeConfig() *rest.Config {
+	config := &rest.Config{
+		Host: "https://1.2.3.4:1234",
+	}
+	return config
+}
+
+func (o *TestKubernetes) NewClustersClientSet() (clustersclientset.Interface, error) {
+	return o.fakeClustersClient, nil
+}
+
+func (o *TestKubernetes) NewVerrazzanoClientSet() (verrazzanoclientset.Interface, error) {
+	return o.fakev8oCleint, nil
+}
+
+func (o *TestKubernetes) NewProjectClientSet() (projectclientset.Interface, error) {
+	return o.fakeProjectClient, nil
+}
+
+func (o *TestKubernetes) NewClientSet() kubernetes.Interface {
+	return o.fakek8sClient
+}
+
+func authHandle(w http.ResponseWriter, r *http.Request) {
+	u, _ := url.Parse(r.URL.String())
```

why suppress errors?